.test-infra/jenkins/jira_utils/jira_manager.py

```diff
@@ -199,20 +209,61 @@ def _find_owners(self, dep_name):
       owners = dep_info['owners']
       if not owners:
         logging.warning("Could not find owners for " + dep_name)
-        return None, []
+        return []
     except KeyError:
       traceback.print_exc()
       logging.warning("Could not find the dependency info of {0} in the OWNERS configurations.".format(dep_name))
-      return None, []
+      return []
     except Exception as e:
       traceback.print_exc()
       logging.error("Failed finding dependency owners: "+ str(e))
-      return None, None
+      return None
 
     logging.info("Found owners of {0}: {1}".format(dep_name, owners))
     owners = owners.split(',')
     owners = map(str.strip, owners)
     owners = list(filter(None, owners))
-    primary = owners[0]
-    del owners[0]
-    return primary, owners
+    return owners
+
+
+  def _need_reopen(self, issue, dep_latest_version):
+    """
+    Return a boolean that indicates whether reopen the closed issue.
+    """
+    # Check if the issue was closed with a "fix version/s"
+    # Reopen the issue if it hits the next release version.
+    next_release_version = self._get_next_release_version()
+    if issue.fields.fixVersions[0].name in next_release_version:
+      return True
+
+    # Check if there is other new versions released.
+    # Reopen the issue if 3 new version has been released in 6 month since closure.
+    try:
+      if issue.fields.resolutiondate:
+        closing_date = datetime.strptime(issue.fields.resolutiondate[:19], "%Y-%m-%dT%H:%M:%S")
+        if (datetime.today() - closing_date).days >= _ISSUE_REOPEN_DAYS:
+          # Extract the previous version when JIRA closed.
+          descriptions = issue.fields.description.splitlines()
+          descriptions = descriptions[len(descriptions)-3]
+          previous_version = descriptions.split("the latest version", 1)[1].strip()
```

Comment:
Could you add a comment on the logic here ? We store the the latest version  in a comment that is in a specific form ? Should we also add a warning to the JIRA to make sure that folks do not modify the comment ?